src/vega.schema.ts

```diff
@@ -28,6 +28,9 @@ import {SortOrder} from './sort';
 
 export {VgSortField, VgUnionSortField, VgCompare, VgTitle, LayoutAlign, ProjectionType, VgExprRef};
 
+/**
+ * @format color-hex
```

Comment:
@domoritz color-hex is not a valid format and breaks the editor! https://json-schema.org/understanding-json-schema/reference/string.html

Reply:
I was going off of https://github.com/microsoft/monaco-json